File: models/Cinema.js
// models/Cinema.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const CinemaSchema = new Schema({
  name:    { type: String, required: true },
  address: String,
  city:    String,
  hotline: String,

  // ✅ Thêm toạ độ
  latitude:  { type: Number },  // vĩ độ
  longitude: { type: Number }   // kinh độ
}, { timestamps: true });

module.exports = mongoose.model('Cinema', CinemaSchema);

File: app/src/main/java/com/example/datn_md_13/Model/Cinema.java
package com.example.datn_md_13.Model;

import java.io.Serializable;

public class Cinema implements Serializable {
    private String _id;
    private String name;
    private String address;
    private String city;
    private String hotline;
    private Double latitude;
    private Double longitude;

    public String getId() { return _id; }
    public String getName() { return name == null ? "" : name; }
    public String getAddress() { return address == null ? "" : address; }
    public String getCity() { return city == null ? "" : city; }
    public String getHotline() { return hotline == null ? "" : hotline; }



    public Double getLatitude()  { return latitude; }
    public Double getLongitude() { return longitude; }

}

